tools/pd-heartbeat-bench/main.go

```diff
@@ -371,39 +410,54 @@ func main() {
 	for i := 1; i <= cfg.StoreCount; i++ {
 		streams[uint64(i)] = createHeartbeatStream(ctx, cfg)
 	}
-	var heartbeatTicker = time.NewTicker(60 * time.Second)
+	var storeUpdateRound = 0
+	var heartbeatTicker = time.NewTicker(storeReportInterval * time.Second)
 	defer heartbeatTicker.Stop()
 	for {
 		select {
 		case <-heartbeatTicker.C:
 			if cfg.Round != 0 && regions.updateRound > cfg.Round {
 				exit(0)
 			}
-			rep := newReport(cfg)
-			r := rep.Stats()
-
-			startTime := time.Now()
+			storeUpdateRound++
 			wg := &sync.WaitGroup{}
-			for i := 1; i <= cfg.StoreCount; i++ {
-				id := uint64(i)
+			// store heartbeat
+			storesStats := regions.collectStoresStats(cfg.StoreCount)
+			for i := 0; i < cfg.StoreCount; i++ {
 				wg.Add(1)
-				go regions.handleRegionHeartbeat(wg, streams[id], id, rep)
+				storeStat := storesStats[i]
+				go func() {
+					defer wg.Done()
+					cli.StoreHeartbeat(ctx, &pdpb.StoreHeartbeatRequest{Header: header(), Stats: storeStat})
+				}()
+			}
+			// region heartbeat
+			if storeUpdateRound%6 == 0 {
+				rep := newReport(cfg)
+				r := rep.Stats()
+				startTime := time.Now()
+				for i := 1; i <= cfg.StoreCount; i++ {
+					id := uint64(i)
+					wg.Add(1)
+					go regions.handleRegionHeartbeat(wg, streams[id], id, rep)
+				}
+				wg.Wait()
```

Comment:
The store heartbeat still needs to wait for the region heartbeat to finish.

Reply:
cc @lhy1024